### PR TITLE
Reduce dio package pana analyzer to 90 points

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Follow up #718

## :bulb: Motivation and Context
_#skip-changelog_

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

## :crystal_ball: Next steps
https://github.com/dart-lang/pana/issues/1020

### CODE REVIEW

File: .github/workflows/dio.yml [hunk 1]
```diff
@@ -84,7 +84,7 @@ jobs:
           TOTAL_MAX: ${{ steps.analysis.outputs.total_max }}
         run: |
           PERCENTAGE=$(( $TOTAL * 100 / $TOTAL_MAX ))
-          if (( $PERCENTAGE < 100 ))
+          if (( $PERCENTAGE < 90 ))
```
Comment (marandaneto) [inline .github/workflows/dio.yml]:
```suggestion
          if (( $PERCENTAGE < 75 ))
```

**Comment by ueman** [comment]
Remember to increase it after publishing the next stable version. Then the issues should be gone.

**Comment by marandaneto** [comment]
> dart-lang/pana#1020

Yep, See this as well: https://github.com/dart-lang/pana/issues/1020

**Review by ueman — APPROVED** [review]
LGTM

**Review by romtsn — APPROVED** [review]

**Pull request merged**